clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -160,7 +164,7 @@ class ClioWebClient(
     retriedResponse.flatMapConcat { response =>
       if (response.status.isSuccess()) {
         logger.debug(s"Successfully completed request: $request")
-        response.entity.withoutSizeLimit().dataBytes.idleTimeout(requestTimeout)
+        response.entity.withoutSizeLimit().dataBytes.idleTimeout(idleTimeout)
```

Comment:
So the code before was using the `requestTimeout` as the `idleTimeout` as well. Why not use the same value for both?

Reply:
Same question